core/src/main/resources/hudson/util/HudsonIsLoading/index_cs.properties

```diff
@@ -0,0 +1,24 @@
+# The MIT License
+#
+# Copyright (c) 2004-2010, Sun Microsystems, Inc.
```

Comment:
I have updated the file replacing header by what is in most of other properties files mentioned regarding to licence. If I should put different text in there, please let me know.